docs/service_x509_credentials_aws.md

```diff
@@ -0,0 +1,346 @@
+![AWS EC2 Setup](images/aws-ec2-setup.png)
+
+**Bootstrapping AWS EC2 instances with Athenz Service x.509 Identities
+requires the instances to have network connectivity to both Athenz ZTS
+and AWS STS Services.**
+
+## Domain Registration
+
+Identify your Athenz domain before you can proceed by using Athenz UI.
+
+You may create a top-level domain or a sub domain using Athenz UI or zms-cli.
+Only Athenz System Administrators can create top level domains (members of the sys.auth role).
+To create a top-level domain using zms-cli run the following:
+
+```
+zms-cli add-domakn <domain-name> <unique-product-id> <domain-admins separated by space>
+```
+
+If you already have top-level domain and need to create a sub domain using Athenz UI:
+click the "Create” link next to "My Domains" label in the top right corner
+and then click on “Sub Domain” tab and follow the on screen instruction.
+
+![Create Sub Domain](images/create_sub_domain.png)
+
+Or using zms-cli:
+```
+zms-cli add-doman <top-doman.sub-domain-name> <domain-admins separated by space>
+```
+
+## AWS Account ID Registration
+
+To register an AWS Account with a domain, run the following command:
+```
+zms-cli -d <domain-name> set-aws-account <aws-account-id>
+```
+
+## Athenz Service Identity Registration
+
+Create a service identity for your AWS EC2 instances in your Athenz domain. This full service identity name `<domain>.<service>` will be the IAM role name that you will need to create in AWS IAM and set up a trust relationship with your EC2 Instance Role.
+
+In the Athenz UI, select your domain, select the `Services` tab and then choose `Add a Service` link in the top left corner. You must provide a service name and an optional description for your service. 
```

Comment:
Removed the reference to bootstrapping (not open source) that said: "Since this is for your AWS EC2 instances, you do not need to provide a public key as the instance will automatically be provisioned with an Athenz Service Identity TLS certificate during bootstrap."